includes/definitions/discovery/unifi.yaml

```diff
@@ -5,3 +5,8 @@ modules:
                 total: FROGFOOT-RESOURCES-MIB::memTotal
                 free: FROGFOOT-RESOURCES-MIB::memFree
                 precision: 1024
+    processors:
+        data:
+            -
+                oid: .1.3.6.1.2.1.25.3.3.1.2
```

Comment:
This is just HOST-RESOURCES-MIB::hrProcessorLoad

Reply:
what if there are more?



.1.3.6.1.2.1.25.3.3.1.2.196608 = INTEGER: 2

.1.3.6.1.2.1.25.3.3.1.2.196609 = INTEGER: 1

.1.3.6.1.2.1.25.3.3.1.2.196610 = INTEGER: 2

.1.3.6.1.2.1.25.3.3.1.2.196611 = INTEGER: 2